modules/post/multi/manage/system_session.rb

```diff
@@ -10,7 +10,7 @@ def initialize(info={})
         'Name'          => 'Multi Manage System Remote TCP Shell Session',
         'Description'   => %q{
           This module will create a Reverse TCP Shell on the target system
-          using the system own scripting enviroments installed on the
+          using the system own scripting environments installed on the
```

Comment:
*system's

Also, I realize it wasn't part of this scope of effort, line 58 has a 'shel' in the output...

Reply:
welp, sorry, meant the following line (13) "using the **system's** own scripting..."  :P